### DISCUSSION TITLE
Custom sort movies inside collection

### DISCUSSION THREAD

**Discussion by bobokun** [meisnate12/Plex-Meta-Manager#86] [Q&A]
If I want to change the order of the movies inside a collection to a custom sort, do I have to do a metadata edit for each movie inside the collection and change the `sort_title`, then set the `collection_order: alpha` for the collection? Or is there a better approach to do this

**Comment by meisnate12** [comment]
right now that's the best approach I could add that ability under collections but what I'm really hoping for is for the actual plex custom collection order you can see more about it on their [forums](https://forums.plex.tv/t/collection-order-custom-new/641518).
